xml/troubleshooting.xml

```diff
@@ -685,12 +685,12 @@
   </sect2>
   
   <sect2 xml:id="sec-trouble-boot-dual-boot">
-   <title> <productname>GRUB 2</productname> fails during reboot on a dual-boot system</title>
+   <title><productname>GRUB 2</productname> fails during reboot on a dual-boot
+    system</title>
    <remark>jufa 2021-06-09: bsc#1185274</remark>
    <para>
-     On dual-boot systems with &opensuse; Leap 15.2 and <productname>Microsoft
-     Windows 10</productname>, disable the <parameter>FASTBOOT</parameter>
-    feature if <productname>GRUB 2</productname> fails during reboot.
+    If <productname>GRUB 2</productname> fails during reboot, disable the
+    <option>Fast Boot</option> setting in the <productname>BIOS</productname>.
```

Comment:
Not sure if `productname` around BIOS is really necessary. I would avoid that as it's not a _specific_ product name, just an acronym:

```suggestion
    <option>Fast Boot</option> setting in the BIOS.
```

Reply:
changed as suggested